source/guides/network/wifi.rst

```diff
@@ -102,8 +104,119 @@ TUI (Text-based User Interface)
 
    .. figure:: /_figures/wifi/nmtui_5.png
 
+wpa_supplicant
+**************
+
+Make sure NetworkManager is either disabled or not installed before using
+wpa_supplicant.
+
+.. code-block:: bash
+
+   sudo systemctl stop NetworkManager.service
+
+#. Create a ``wpa_supplicant`` configuration directory.
+
+   .. code-block:: bash
+
+      sudo mkdir /etc/wpa_supplicant
+
+#. Create a minimal :file:`/etc/wpa_supplicant/wpa_supplicant.conf` and add the
```

Comment:
Suggest making this `/etc/wpa_supplicant/wpa_supplicant-$INTERFACE_NAME.conf`. Reasoning in a comment below. This means the step to determine wireless interface needs to come above.

Reply:
I suggest making explicit what to do with the interface name after run `iw dev`. If the user should see this interface name used after running command on Ln 142, then that should be stated. 
FYI only: In testing, I successfully replaced `$INTERFACE` with the name resulting from `iw dev`.

Reply:
@puneetse, Are you sure that you want to use the `-` before $INTERFACE_NAME.conf? 
As in: `/etc/wpa_supplicant/wpa_supplicant-$INTERFACE_NAME.conf`

Reply:
Yes, see the reason below. It's a distro-distributed systemd unit template which should be leveraged.